### ISSUE TITLE
Update script

### ISSUE THREAD

**Issue by markesmith** [vitorhirota/tcx-edit#1]
Hello @vitorhirota 

I spent a good few hours searching the internet for this exact script. However, I could not get it to run. I realise this script is quite old and has had no activity but I wanted to reach out to you just in case you were in a position to take a look at it and get it working in 2020. 

I ran the script and worked through some syntax errors I received, such as parenthesis in print functions and missing lxml libraries. However, I have reached the end of my skills (hardly any scripting skills at all) and am now stuck. 

The last errors I received in CMD are below: 
```
C:\tcx-edit-master>python tcx-edit.py C:\tcx-edit-master\activity.tcx merge 11-12
## TCX Edit ##
C:\tcx-edit-master\activity.tcx: merge with ['11-12'] arguments
Traceback (most recent call last):
  File "C:\tcx-edit-master\tcx-edit.py", line 160, in <module>
    fn(args.tcx_file, args.arguments)
  File "C:\tcx-edit-master\tcx-edit.py", line 112, in merge
    track.append(track.find('./ts:Extensions', namespaces=ns))
TypeError: Argument 'element' has incorrect type (expected lxml.etree._Element, got NoneType)
```
Any help would be greatly received, best regards MES.

**Comment by vitorhirota** [comment]
Hello @markesmith 

Glad to know it's useful for someone other than myself :)

This is written for python 2, thus the parenthesis complain for print function. Back then I totally missed to also register what version of lxml I was using. So you might want to try [3.4.x](https://lxml.de/3.4/) since it's from around 2014.

Let me know if that works out.

**Comment by markesmith** [comment]
HI @vitorhirota 

It's very kind of you to reply to such an old script. Thank-you. 

However, despite some further playing around I think I am going to have to admit defeat - at least until I understand a little more. 

I tried to install an older version of lxml but I received many errors, I Googled some more and discovered I might also need libxml2 and libslt but despite adding their respective bin folders to my env variables I was still unsuccessful in installing an older version of lxml.  

So, I will return to this topic once I understand things a little more. Thanks once again for taking the time to reply and best regards, MES.

**Comment by vitorhirota** [comment]
> However, despite some further playing around I think I am going to have to admit defeat - at least until I understand a little more.

It was a brave attempt, I'm sure :)

If you want to share your .tcx, I will fiddle around to see what I can find out.

One way to do that, is to create a private gist with your .tcx contents, and just mark me in the comments.

**Comment by vitorhirota** [comment]
@markesmith just comment line 112 and re-execute. 

As far as I could check, TomTom generates an Extension tag at the end of each Track, whereas Garmin puts that Extension outside Track, thus the error.

**Comment by markesmith** [comment]
Hi @vitorhirota 

Thanks for the reply and pointer towards line 112. I commented this line out but still retuned an error which pointed to line 129. I commented this line out and hey presto I got a file which has indeed merged the laps I wanted: 

![image](https://user-images.githubusercontent.com/64022365/97096055-ba1afd80-165e-11eb-9271-8026173e35f8.png)

I am guessing line 129 refers to the max HR value which has returned zero for the merged laps: 

`#sum(hrm)/len(hrm),`

I am not too concerned about max HR as average HR for the lap is more relevant to me anyway. 

Thank you so much for taking time out to help out with getting me to this point. 

Best regards,
